### PR TITLE
Modify rule S3403: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3403/python/rule.adoc [hunk 1]
```diff
@@ -1,16 +1,36 @@
+This rule raises an issue when an identity comparison operator is used to compare objects of different types.
+
 == Why is this an issue?
 
-Operators https://docs.python.org/3/reference/expressions.html#is-not[``++is++``] and https://docs.python.org/3/reference/expressions.html#is-not[``++is not++``] check if their operands point to the same instance, thus they will always return respectively True and False when they are used to compare objects of different type. Such comparisons can only be bugs.
+Operators https://docs.python.org/3/reference/expressions.html#is-not[``++is++``] and https://docs.python.org/3/reference/expressions.html#is-not[``++is not++``] check if their operands point to the same instance, thus they will always return respectively True and False when they are used to compare objects of different types. Such comparisons can only be bugs.
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S3403/python/rule.adoc:5]: `is` will return False, and `is not` will return True, right?
I think we should reverse the `True` and `False` in the sentence to be consistent.

**Review by guillaume-dequenne-sonarsource — CHANGES_REQUESTED** [review]
Looks good, minor change request.

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM.

File: rules/S3403/python/rule.adoc [hunk 2]
```diff
@@ -1,16 +1,36 @@
+This rule raises an issue when an identity comparison operator is used to compare objects of different types.
+
 == Why is this an issue?
 
-Operators https://docs.python.org/3/reference/expressions.html#is-not[``++is++``] and https://docs.python.org/3/reference/expressions.html#is-not[``++is not++``] check if their operands point to the same instance, thus they will always return respectively True and False when they are used to compare objects of different type. Such comparisons can only be bugs.
```
Comment (jean-jimbo-sonarsource) [inline rules/S3403/python/rule.adoc:3]: " Such comparisons can only be bugs." sounds like a filler - it's not actually giving useful information. I'd suggest discarding it as the reasoning has already been given and it is an issue/bug.

**Review by jean-jimbo-sonarsource — APPROVED** [review]
Added a small feedback

**Pull request merged**